packages/garbo/src/tasks/cockroachPrep.ts

```diff
@@ -253,12 +316,10 @@ export const CockroachSetup: Quest<GarboTask> = {
       },
       outfit: () => {
         if (get("_lastPirateRealmIsland") === $location`Crab Island`) {
-          const spec: OutfitSpec = {
+          const spec = meatTargetOutfit({
             modifier: ["meat"],
             equip: $items`PirateRealm eyepatch`,
-            avoid: $items`cursed pirate cutlass`, // Gives +25 muscle which often overcaps
```

Comment:
seems like you're cutting out helpful and load-bearing lines without replacing them with anything

Reply:
Helpful maybe, but hard coding them is inappropriate in this case. While we should value the red roger equips, avoiding the cursed pirate cutlass is strictly unnecessary when we're managing buffs more effectively.